src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/Algebras/BVAlgebra.cs

```diff
@@ -247,7 +184,23 @@ public string PrettyPrint(BV bv)
             ICharAlgebra<BDD> bddalgebra = SymbolicRegexRunner.s_unicode._solver;
 
```

Comment:
As in the `ulong` case, the following should work now:
`public string PrettyPrint(BV bv) => CharSetSolver.PrettyPrint(ConvertToCharSet(CharSetSolver, bv));`

Reply:
Thanks, changed.